requirements.txt

```diff
@@ -4,7 +4,7 @@ pyparsing
 ipython
 pandas
 tqdm
-networkx
+networkx<2.8.3
```

Comment:
we should actually fix the problem with the new version of networkx.

Reply:
After reviewing `networkx`'s approach, we determined that they made python `3.8` a requirement in later releases before `3.7` has reached EOL. We'll pin for now until we are ready to make `3.8` our minimum version.